src/components/Link.tsx

```diff
@@ -35,6 +37,7 @@ function Link(props: IProps) {
 
   return (
     <a
+      {...props}
```

Comment:
The problem is `...props` add `to` prop too. 
```html
<a to="[object Object]" target="_blank" class="Link active" href="/base/en">Home</a>
```
I can exclude it but, maybe there is a better solution ?

Reply:
I can only think of other solutions which would introduce too much logic in this component.
Updated the PR with exclusion changes